index.js

```diff
@@ -128,6 +128,12 @@ module.exports = {
     "no-undef": "off",
     "no-unused-vars": "off",
 
+    // emotion
+    "emotion/jsx-import": "error",
+    "emotion/no-vanilla": "error",
+    "emotion/import-from-emotion": "error",
+    "emotion/styled-import": "error",
+
```

Comment:
Should we add the `emotion/syntax-preference: [2, "string"]` rule ? So that we enforce only one syntax ?